sql/hive/src/test/resources/ql/src/test/queries/clientpositive/smb_mapjoin_25.q

```diff
@@ -19,7 +19,7 @@ select * from (select a.key from smb_bucket_1 a join smb_bucket_2 b on (a.key =
 
 set hive.optimize.bucketmapjoin=true;
 set hive.optimize.bucketmapjoin.sortedmerge=true;
-set hive.mapred.reduce.tasks.speculative.execution=false;
+set hive.mapreduce.job.reduces.speculative.execution=false;
```

Comment:
Is this supposed to be `mapreduce.reduce.speculative`? I'm looking at https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/DeprecatedProperties.html
Or maybe the hive.* version is different?